rust/shalom_dart_codegen/templates/operation.dart.jinja

```diff
@@ -1,91 +1,95 @@
-{% macro base_object_impl(typename, selections) %}
-
-/// class memberes
-{% for selection in selections %}
-final {{type_name_for_selection(selection)}} {{selection.selection_name}};
-
-{% endfor %}
-
-// keywordargs constructor
-{{typename}}({
+{% macro generate_class(typename, selections, root, is_root=false) %}
```

Comment:
it is intentional that I used `base_object_impl` and not generated the whole class here. we might be using this macro for other stuff in the future.
revert.